libgadget/gravshort.h

```diff
@@ -43,13 +43,19 @@ struct GravShortPriv {
     int NeutrinoTracer;
     /* Newton's constant in internal units*/
     double G;
+    inttime_t Ti_Current;
     /* Matter density in internal units.
      * rho_0 = Omega0 * rho_crit
      * rho_crit = 3 H^2 /(8 pi G).
      * This is (rho_0)^(1/3) ,
      * Note: should account for
      * massive neutrinos, but doesn't. */
     double cbrtrho0;
+    /* Whether to calculate the gravitational potential.
+     * Note this is only useful on steps where all particles are active*/
+    int CalcPotential;
```

Comment:
The comment is a bit confusing -- you meant CalcPotential is True only on steps where all particles are active?

If so, perhaps rename this to AllParticlesAreActive?

Reply:
I reworded the comment: it is not quite AllParticlesAreActive because we still need the potential when hierarchical gravity is off.